File: C/palindrome.c
#include<stdio.h>
#include<string.h>
int main(){
    int len,i;
    char str[15];
    printf("Enter the string : ");
    scanf("%s",str);
    len = strlen(str);
    for(i=0;i<len;i++){
        if(str[i]==str[len-i-1]){
            printf("The entered word %s is a palindrome",str);
            break;
        }
        else{
            printf("The word %s is not a palindrome",str);
            break;
        }
    }
    return 0;
}

File: Java/src/com/vaishakh/Basic/palindrome.java
package com.vaishakh.Basic;

import java.util.Scanner;

public class palindrome {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter the word to be checked for palindrome");
        String word = in.nextLine();
        int x = word.length();
        char[] letters=word.toCharArray();
        for (int i=0;i<=x;i++) {
            if (letters[i]==letters[x-i-1]){
                System.out.println(word+ " is a palindrome");
                break;
            }
            else {
                System.out.println(word+ " is not a palindrome");
            }
        }
    }
}
